libcloud/compute/drivers/cloudsigma.py

```diff
@@ -984,10 +984,10 @@ class CloudSigma_2_0_NodeDriver(CloudSigmaNodeDriver):
     NODE_STATE_MAP = {
         'starting': NodeState.PENDING,
         'stopping': NodeState.PENDING,
-        'unavailable': NodeState.PENDING,
```

Comment:
@vpetersson Is mapping "unavailable" to "error" indeed more correct than mapping it to "pending" - aka can node come out of the "unavailable" state?